src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportMakerService;
+import core.basesyntax.service.TransactionParserService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportMakerServiceImpl;
+import core.basesyntax.service.impl.TransactionParserServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.ActionStrategy;
+import core.basesyntax.strategy.actions.ActionHandler;
+import core.basesyntax.strategy.actions.BaseActionHandler;
+import core.basesyntax.strategy.actions.PurchaseActionHandler;
+import core.basesyntax.strategy.actions.ReturnActionHandler;
+import core.basesyntax.strategy.actions.SupplyActionHandler;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    private static final Storage storage = new Storage();
+    private static final String FROM = "src/main/resources/test.csv";
+    private static final String TO = "src/main/resources/";
+
+    public static void main(String[] args) {
+        ActionStrategy strategy = new ActionStrategy(allPossibleActions());
+        ReaderService reader = new ReaderServiceImpl();
+        WriterService writer = new WriterServiceImpl();
+        TransactionParserService parser = new TransactionParserServiceImpl();
+        ReportMakerService reportMaker = new ReportMakerServiceImpl();
```

Comment:
let's initialize our services as a class-level fields, and you can use `Map.of()` method to initialize a map for strategy instead of separate method